From 62c922f43b54585d25ba89be36c10612b479270b Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 27 Apr 2017 02:15:02 -0400
Subject: [PATCH] configure.ac: remove prog test of augparse

The build time does not require it.

Upstream-Status: Inappropriate [workaround]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Modify to apply to v2.17.8.

Signed-off-by: Joe Slater <joe.slater@windriver.com>
---
 configure.ac | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/configure.ac b/configure.ac
index 17ece200..89b463a4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -287,15 +287,6 @@ AC_ARG_WITH(largedatatmpdir,
             [], [with_largedatatmpdir="/var/tmp"])
 AC_SUBST([LARGE_DATA_TMP_DIR], [$with_largedatatmpdir])
 
-AC_PATH_PROG(AUGPARSE, augparse, no)
-[if test "$AUGPARSE" = "no"]
-[then]
-    [echo "The augparse program was not found in the search path. Please ensure"]
-    [echo "that it is installed and its directory is included in the search path."]
-    [echo "Then run configure again before attempting to build libreport."]
-    [exit 1]
-[fi]
-
 AC_ARG_WITH([defaultdumpdirmode],
             AS_HELP_STRING([--with-defaultdumpdirmode=OCTAL-MODE],
                            [Default dump dir mode (default: 0x1A0 (0640))]),
-- 
2.25.1

